src/software/ai/hl/stp/tactic/test_tactics/move_test_tactic.cpp

```diff
@@ -38,3 +38,9 @@ void MoveTestTactic::calculateNextIntent(IntentCoroutine::push_type &yield)
                                            Angle::zero(), 0.0, 0));
     } while ((this->robot->position() - this->destination).len() > 0.01);
 }
+
+void MoveTestTactic::accept(TacticVisitor &visitor) const
+{
+    // WARNING: Tactic Vistor does not implement visiting this Tactic
+    assert(false);
```

Comment:
Generally we prefer exceptions to assertions (for example, because you can actually handle them if you so choose). Could we find an appropriate exception to use here instead?

Reply:
done